configs/x86/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: x86
```

Comment:
```suggestion
  name: x86_64
```

Reply:
I thought about it.  I would need to rename the directory as well - and chance the CI (since that's the only system we are currently testing).